app/consumer.py

```diff
@@ -50,7 +52,8 @@ def on_message(self, unused_channel, basic_deliver, properties, body):
             queue=self.QUEUE,
             delivery_tag=basic_deliver.delivery_tag,
             delivery_count=delivery_count,
-            app_id=properties.app_id
+            app_id=properties.app_id,
```

Comment:
Is properties.app_id always guaranteed to exist? If not this will raise an AttributeError and cause the app to fail.